sound/soc/sof/intel/byt.c

```diff
@@ -510,7 +510,7 @@ static struct snd_soc_dai_driver byt_dai[] = {
 
 #if IS_ENABLED(CONFIG_SND_SOC_SOF_MERRIFIELD)
 
-static int byt_pci_probe(struct snd_sof_dev *sdev)
+static int tangier_pci_probe(struct snd_sof_dev *sdev)
```

Comment:
Or `tng`, depends which one is consistent with the rest.
I'm fine for either, but see above.